.github/workflows/integration_tests.yaml

```diff
@@ -73,7 +77,10 @@ jobs:
         run: kubemarine check_paas -c ${{ env.cluster_yaml_file }} --dump-location ./results/check_paas_dump/
       - name: Change not-recommended symbols in dump files
         if: failure()
-        run: rename 's/[:]/_/g' ./results/*/dump/*
+        run: |
+          sudo journalctl -u kubelet -u kube-proxy -u kube-apiserver -u kube-controller-manager -u kube-scheduler > ./results/journald_kube-logs.txt
+          sudo journalctl > ./results/journald-logs.txt 
```

Comment:
Please, move collecting journals to separate step, because this one is for another case;
And add the same logic to `test-extended-installation` job below.

Reply:
okay